activejob/lib/active_job/core.rb

```diff
@@ -137,6 +141,11 @@ def deserialize(job_data)
       self.exception_executions = job_data["exception_executions"]
       self.locale               = job_data["locale"] || I18n.locale.to_s
       self.timezone             = job_data["timezone"] || Time.zone.try(:name)
+      self.enqueued_at          = job_data["enqueued_at"]
+    end
+
+    def enqueded_for
+      Time.now.utc - enqueued_at
```

Comment:
Is this working? `enqueued_at` will be a string and when I try `Time.now.utc - JSON.load(JSON.dump(Time.now.utc))` I get an error.

Reply:
I think I will remove it per @tenderlove's suggestion.